linux-kernel-headers Bug#263348: linux-kernel-headers: asm/unaligned.h on powerpc is useless if __KERNEL__ is undefined

This patch removes unneeded #ifdef __KERNEL__ from ppc unaligned.h.
There's no reason to enclose __KERNEL__, similar to other
architectures.  It also fixes the compilation failure building
reiserfsprogs on ppc which includes asm/unaligned.h on userland.


--- include/asm-ppc/unaligned.h	2003-12-15 22:57:40.000000000 +0900
+++ include/asm-ppc/unaligned.h.new	2004-08-09 12:17:16.000000000 +0900
@@ -1,4 +1,3 @@
-#ifdef __KERNEL__
 #ifndef __PPC_UNALIGNED_H
 #define __PPC_UNALIGNED_H
 
@@ -15,4 +14,3 @@
 #define put_unaligned(val, ptr) ((void)( *(ptr) = (val) ))
 
 #endif
-#endif /* __KERNEL__ */
